packages/angular/angular-sdk/src/lib/sdk/space.component.ts

```
@@ -36,98 +37,21 @@
     if (!this.spaceProps) throw new Error('Please define the space props')
 
     if (this.openDirectly) {
-      await this.initSpace(this.spaceProps)
-    } else {
-      this.appService.signal.subscribe(async () => {
-        await this.initSpace(this.spaceProps)
-      })
+      await this.appService.initSpace(this.spaceProps)
     }
+    const self = this
```

Biome

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)